AppController/lib/taskqueue.rb

```diff
@@ -108,6 +105,12 @@ def self.start_master(clear_data, verbose)
     # First, start up RabbitMQ.
     start_rabbitmq
 
+    # The master rabbitmq will set the policy for replicating the messages
+    # on the queues.
+    cmd = "#{RABBITMQCTL} set_policy ha-all "
+    cmd += "\"\" '{\"ha-mode\":\"all\", \"ha-sync-mode\": \"automatic\"}'"
```

Comment:
I would recommend either using `%Q[string]` or building the hash in ruby and then doing a json serialization. All the escape characters makes it more difficult to read.